packages/cli/README.md

```diff
@@ -14,23 +14,23 @@ npm i @basemaps/cli
 
 ## Usage -- Bundle
 
-Bundle config files into config bundle json from a given config path.
+Bundle config files into config bundle json from a given config path. This is mainly use for [@basemaps-config](https://github.com/linz/basemaps-config) ci/cd process to bundle all the individual config json files into one.
 
 ```bash
-./bin/bmc.js bundle --config config/ --output config.json
+./bin/bmc.js bundle --config config/ --assets assets/asset.tar --output config.json
 ```
 
 ## Usage -- Import
 
-Import all configs from a bundled config.json into dynamo db from a given config path
+Import all configs from a bundled config.json into dynamo db from a given config path. This is mainly use for [linz/basemaps-config](https://github.com/linz/basemaps-config) repository ci/cd process to deploy the config changes into DynamoDB.
 
 ```bash
 ./bin/bmc.js import --config config.json --commit
 ```
 
 ## Usage -- Screenshots
 
-Dump the screenshots from basemaps production
+Dump the screenshots from basemaps production, this mainly using for the screenshot validation smoke tests in the basemaps ci/cd process.
 
 ```bash
 ./bin/bmc.js screenshot
```

Comment:
I thought we moved this into linz/basemaps-screenshot?

Reply:
You are correct, I will remove this.